### PR TITLE
Add 'change history' styles to Style Configurator for theme without them

### PR DESCRIPTION
**Pull request by ozone10**
ref #14883

### CODE REVIEW

**Comment by ozone10** [comment]
Splitted from https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14883.

also check
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/12c6777bc11b47074b7d9848eac48a91c49b2594/PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp#L2902
for other 'default' styles

@donho 
I've noticed little inconsistency, colors from 
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/12c6777bc11b47074b7d9848eac48a91c49b2594/PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp#L2902
for eol and npc are not same, `DADADA`  and `liteGrey` (`C0C0C0`)

**Comment by donho** [comment]
@ozone10 
> I've noticed little inconsistency, colors from
...
for eol and npc are not same, DADADA and liteGrey (C0C0C0)

Why should both of them should be the same color?

**Comment by ozone10** [comment]
@donho 
>Why should NPC & EOL should be the same color?

They already have same colors, just in
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/b31e7a31e099f6ecd97b12c1b9053b0537a69acc/PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp#L3097
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/b31e7a31e099f6ecd97b12c1b9053b0537a69acc/PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp#L3105
they use `liteGrey` which is `C0C0C0`
but in
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/b31e7a31e099f6ecd97b12c1b9053b0537a69acc/PowerEditor/src/Parameters.cpp#L4164
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/b31e7a31e099f6ecd97b12c1b9053b0537a69acc/PowerEditor/src/Parameters.cpp#L4187
they use `DADADA`

File: PowerEditor/src/Parameters.cpp [hunk 1]
```diff
@@ -4155,40 +4155,73 @@ bool NppParameters::feedStylerArray(TiXmlNode *node)
...
+			return result;
+		};
+
+	addStyle(L"Change History modified", L"FF8000", L"FF8000");
+	addStyle(L"Change History revert modified", L"A0C000", L"A0C000");
+	addStyle(L"Change History revert origin", L"40A0BF", L"40A0BF");
+	addStyle(L"Change History saved", L"00A000", L"00A000");
+
+	addStyle(L"EOL custom color", L"DADADA");
+	addStyle(g_npcStyleName, L"DADADA", L"", L"White space symbol");
```
[the remark this answers is not in the document]
Comment (ozone10) [reply PowerEditor/src/Parameters.cpp:4224]: it was from previous commit and theme usually already have white space symbol style defined and NPC are mostly some kind of "unicode spaces", so they use same default color

[the remark this answers is not in the document]
Comment (donho) [reply PowerEditor/src/Parameters.cpp:4224]: Yes, you're right. Sorry for not seeing it.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**